src/utils/actions.js

```diff
@@ -0,0 +1,139 @@
+const actions = [
+  {
+    value: 'applyBookmark',
+    label: 'Apply a bookmark',
+    group: 'bookmark',
+    promise: ({ engineApp, bookmark }) => async () => {
+      await engineApp.applyBookmark(bookmark);
+    },
+    requiredInput: ['bookmark'],
+  },
+  {
+    value: 'clearAll',
+    label: 'Clear all selections',
+    group: 'selection',
+    promise: ({ engineApp }) => async () => {
+      await engineApp.clearAll();
+    },
+    requiredInput: [],
+  },
+  {
+    value: 'clearOther',
+    label: 'Clear selections in other fields',
+    group: 'selection',
+    promise: ({ engineApp, field, softLock }) => async () => {
+      const fld = await engineApp.getField(field);
```

Comment:
`fld` as in field? could have a better variable name, like `fieldObj` or something

Reply:
Ah yes, that sounds a lot better. Thanks :)